include/wx/qt/statusbar.h

```diff
@@ -45,7 +47,7 @@ class WXDLLIMPEXP_CORE wxStatusBar : public wxStatusBarBase
     void UpdateFields();
 
     QStatusBar *m_qtStatusBar;
-    QList< QLabel* > *m_qtPanes;
+    QScopedPointer< QList<QLabel*> > m_qtPanes; // should this really be a pointer?
```

Comment:
It really doesn't look like it should... And I'd rather use (just) `wxVector<QLabel*>` than `QList` but this is just me.